### ISSUE TITLE
ASUS Aura Sync Version Issue

### ISSUE THREAD

**Issue by KyleStilkey** [AndrewBabbitt97/AuraConnect#6]
I'm not sure what version you are suppose to be using to make this function, but with the latest version 1.07.71 is currently broken with the latest version of Windows 1909 (Fall 2019 Update).

Version 1.06.17 does show up under Synapse but shows up as unavailable. This version does work with Windows 1909 and is their latest recommended version to use. If this version does work with AuraConnect it's not on my end, but if it's not intended to, is there a chance you will update the software to support their latest recommended version?

I see on the website they don't even list 1.06.17 anymore but based on their Forums it's what they recommend using still. https://rog.asus.com/forum/showthread.php?101543-SOFTWARE-AURA-Sync-RGB

I will test the other 2 versions they have listed 1.07.60 and .66 and see if there is any luck or not.

**Comment by KyleStilkey** [comment]
Seems like whatever changes they made to the 7.xx branch is not functioning under Windows 1909.

**Comment by AndrewBabbitt97** [comment]
I will look into this.

1909 is not GA as far as I'm aware, my development environment is based on 1903 and is working fine.

There is research work to be done in terms of the earliest available Aura version that can be used. ASUS changed Aura to use HALs and added a COM component to be used as an SDK.

Finding the first version that used this approach is definitely on the to do list, however there is not much I can do if 1909 has issues with the latest versions of AURA aside from waiting for ASUS to release a new version.

Please update me on any progress you have on getting other versions to work, id like to document it.

**Comment by AndrewBabbitt97** [comment]
I am using 1.07.71 on my PC running 1903. If it's not working on 1909 then it will be up to ASUS to fix. There isn't anything I can really do on my side to fix it.

**Comment by KyleStilkey** [comment]
I updated above, just checking with the forums a lot of users are having issues with their 7.xx branch and they have noted on the forums there is known issues and no ETA on any fixes leaving a lot of users stuck on 1.06.17.

If it's possible to add support to this latest recommended version that would be awesome. As far as when they made the changes over to the new HALs/COM was with the version release 7.xx (They released a lot of pretest versions on their Forums.

**Comment by KyleStilkey** [comment]
Oh I know you can't do anything on their software, just wanted to see if it's possible to added support to their last recommended version that a lot of users are left on.

**Comment by AndrewBabbitt97** [comment]
I'm going to guess that 1.06.x versions are using the old Aura SDK which is not supported. It may be possible for me to add support for these old versions of Aura but it's something that will take some time to implement.

**Comment by KyleStilkey** [comment]
Ok thanks for the update, when ASUS releases a fixed version I will report back here and let you know. I appreciate the response and await to finally have my RGB setup completed.

**Comment by KyleStilkey** [comment]
Oh and by chance, would it be possible to change the ReadMe to state that the version required to use this is 1.07.71 so others know this too? As by the ASUS saying 1.06.17 is the latest recommended version, might help some other users out too.

**Comment by AndrewBabbitt97** [comment]
I'm doing some changes overall to the structure of the project right now to allow for expandability, and things may change around a bit but this will be documented soon.

**Comment by KyleStilkey** [comment]
Ok here is something weird, Aura still doesn't work but whatever reason, the Aura Service is running and it's being detected by my Synapse. I have no idea why but it's just... working...

I'm guessing if the services can at least start that's all that's required to run this. I will still update this once ASUS releases a working version for the newest Windows build I will still update this so others can see and you are updated as well.

So it seems so long as the services can start, it should work. I spent the last few hours trying to get this to work. Well I appreciate the great help and support you have brought.

**Comment by AndrewBabbitt97** [comment]
I feel like this is something others have been running into...

We're you trying to open the AURA application itself? Like the UI for changing effects?

**Comment by KyleStilkey** [comment]
I was trying to open it but it crashes open starting up, but Synapse just would not see it. I went back to 1.06.17 and Synapse could see it and AURA works but would not function with Synapse as it said it was unavailable. So I upgraded back to 1.07.71 and it crashed out as usual but the services seems to still be running and now Synapse is controlling it without any issues.

**Comment by AndrewBabbitt97** [comment]
So as a part of documentation I have been working on:

This application is a service that runs in the background. It communicates with Aura and Synapse and bridges the gap between the two.

I would recommend against trying to use the aura application at all, the services are all that's needed for this to work. Even if you could open the aura application I could not say how it would act, AuraConnect gains exclusive control over Aura and controls all the RGB lights.

Synapse should be used to control Aura devices after this is installed.

**Comment by KyleStilkey** [comment]
Yeah I know Synapse needs to control it, just like with other connect apps (I have thermaltake as one too) just before the services part wouldn't even run and is why I reported this originally due to their recent bug, but I'm guessing whatever it was that I did at least got the services part working.

With 1.07.xx they did release the services as their own install, and I will pray for the day that they just release that as it's own for this exact reason.

I have many devices that have ASUS AURA support, I will continue to test across the board and if anything new shall arise I will be sure to post a new topic on here or if anything changes I will update this page as well.

Keep up the great work man!
